recipe/meta.yaml

```diff
@@ -18,50 +18,97 @@ build:
     - wcslint = astropy.wcs.wcslint:main
-  number: 1
+  number: 3
   script: {{ PYTHON }} -m pip install . -vv
   skip: true  # [py<310]
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython >=3.0.0,<3.1.0                          # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - {{ compiler('c') }}
-    - {{ stdlib("c") }}
   host:
     - python
-    - pip
-    - setuptools
-    - jinja2 >=3
-    - cython >=3.0.0,<3.1.0
-    - setuptools_scm >=6.2
-    - extension-helpers =1
-    - numpy
-  run:
-    - python
-    - numpy >=1.23
-    - pyerfa >=2.0.1.1
-    - astropy-iers-data >=0.2024.7.29.0.32.7
-    - importlib-metadata
-    - pyyaml >=3.13
-    - packaging >=19.0
 
-test:
-  requires:
-    - pytest-astropy
-  commands:
-    - fits2bitmap --help
-    - fitscheck --help
-    - fitsdiff --help
-    - fitsheader --help
-    - fitsinfo --help
-    - samp_hub --help
-    - volint --help
-    - wcslint --help
-  imports:
-    - astropy
+outputs:
+  - name: astropy-base
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - cython >=3.0.0,<3.1.0                  # [build_platform != target_platform]
+        - numpy                                  # [build_platform != target_platform]
+        - {{ compiler('c') }}
+        - {{ stdlib("c") }}
+      host:
+        - python
+        - pip
+        - setuptools
+        - jinja2 >=3
+        - cython >=3.0.0,<3.1.0
+        - setuptools_scm >=6.2
+        - extension-helpers =1
+        - numpy
+      run:
+        - python
+        - numpy >=1.23
+        - pyerfa >=2.0.1.1
+        - astropy-iers-data >=0.2024.7.29.0.32.7
+        - importlib-metadata
+        - pyyaml >=3.13
+        - packaging >=19.0
+        # Recommended deps
+        - scipy >=1.8
+        - matplotlib-base >=3.5.0,!=3.5.2
+
+    test:
+      requires:
+        - pytest-astropy
+      commands:
+        - fits2bitmap --help
+        - fitscheck --help
+        - fitsdiff --help
+        - fitsheader --help
+        - fitsinfo --help
+        - samp_hub --help
+        - volint --help
+        - wcslint --help
+      imports:
+        - astropy
+
+  - name: astropy
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('astropy-base', max_pin="x.x.x") }}
+        - certifi
+        - dask[array] >=2022.8.1
+        - h5py
+        - pyarrow >=10.0.1
+        - beautifulsoup4
+        - html5lib
+        - bleach
+        - pandas >=2.0
+        - sortedcontainers
+        - pytz
+        - jplephem
+        - mpmath
+        - asdf-astropy >=0.3
```

Comment:
Yeah, I am unconvinced this *whole* list needs to be here, was focusing on getting it working rn.

Reply:
I suspect it would be possible to specify a different python version skip for the two variants. We might need to ask one of the :mage: about how they handle this. Based on the conda-forge docs: https://conda-forge.org/docs/maintainer/knowledge_base/#multi-output-recipes there are quite a number of packages doing this, so I suspect there's an answer to this.

Reply:
I do think it's worth taking a fair few lumps to make this work though, the status quo is frankly [shit](https://github.com/astropy/astropy-APEs/pull/86#issuecomment-2343835171)

Reply:
According to helpful people in the chat: https://matrix.to/#/%23conda-forge%3Amatrix.org/%241726070516272dQqES%3Agitter.im?via=matrix.org&via=gitter.im&via=ifisc.uib-csic.es

It would block until all optional deps are migrated. A suggestion from the chat is to make two feedstocks.

Reply:
Can two feedstocks both receive automated bot PRs triggered by the same pypi release? If yes, then 👍 on that approach from me.

Reply:
I assume the tick bot works on the recipie alone, so it would work? Shall I make a PR to staged recipes for astropy-base?